src/assets/locales/en/translation.json

```diff
@@ -46,7 +46,7 @@
   "update": "Update",
   "collateralization": "Collateralization",
   "fees_earned": "Fees earned",
-  "apy": "APY",
+  "apy": "APR",
```

Comment:
I think we should change the locale text key to `"apr"` to keep the codebase clear.